modular_skyrat/modules/neural-lacing/code/implanter.dm

```diff
@@ -9,8 +9,8 @@
 	invasive = 0
 
 /obj/item/autosurgeon/organ/syndicate/corticalstack
-	desc = "A single use autosurgeon that contains a cortical stack. A screwdriver can be used to remove it, but implants can't be placed back in."
-	uses = 1
+	desc = "A single use autosurgeon that contains a blood-red cortical stack. This implanter is modified to be usable twice."
+	uses = 2
```

Comment:
single-use
2 uses

Reply:
Didnt I already  do this? Locally it says this correctly

Reply:
![image](https://user-images.githubusercontent.com/65188584/124083250-8ee67a80-da45-11eb-837d-91928fb528c8.png)

Reply:
That is ... not correct. The description says "single use", yet it has two uses.

Reply:
I am indeed deranged, and wrong